src/MLJNaiveBayesInterface.jl

```diff
@@ -19,48 +20,45 @@ end
 function MMI.fit(model::GaussianNBClassifier, verbosity::Int
                 , X
                 , y)
-
-    Xmatrix = MMI.matrix(X)' |> collect
+    Xmatrix = MMI.matrix(X) |> permutedims
     p = size(Xmatrix, 1)
 
-    yplain = Any[y...] # y as Vector
+    yplain = convert(Vector, y) # y as Vector
     classes_seen = unique(yplain)
 
     # initiates dictionaries keyed on classes_seen:
     res = NaiveBayes.GaussianNB(classes_seen, p)
 
-    fitresult = NaiveBayes.fit(res, Xmatrix, yplain)
+    fitresult = NaiveBayes.fit(res, convert(Matrix{Float64}, Xmatrix), yplain)
 
     report = NamedTuple{}()
 
     return fitresult, nothing, report
 
 end
 
-function MMI.fitted_params(model::GaussianNBClassifier, fitresult)
-    res = fitresult[1]
-    return (c_counts=res.c_counts,
-            c_stats=res.c_stats,
-            gaussians=res.gaussians,
-            n_obs=res.n_obs)
+function MMI.fitted_params(::GaussianNBClassifier, fitresult)
+    return (c_counts=fitresult.c_counts,
+            c_stats=fitresult.c_stats,
+            gaussians=fitresult.gaussians,
+            n_obs=fitresult.n_obs)
 end
 
 function MMI.predict(model::GaussianNBClassifier, fitresult, Xnew)
+    Xmatrix = MMI.matrix(Xnew) |> permutedims
```

Comment:
Could you comment on this choice over the adjoint? The idea here was to allow a savy user to avoid transpose by wrapping his observation-as-rows matrix `A` as `Tables.matrix(A')`.

Reply:
`collect` will create a copy anyway, so the current implementation did not seem to avoid any allocations and it felt more natural to use `permutedims` instead of `' |> collect`. Additionally, the recursive application of `'` in contrast to `permutedims` seemed unnecessary. Last but not least, it was inconsistent with the implementation for `MultinomialNBClassifier` which already uses `permutedims`.